src/utils/body.ts

```diff
@@ -27,26 +25,18 @@ export const parseBody = async <T extends BodyData = BodyData>(
   options: ParseBodyOptions = { all: false }
 ): Promise<T> => {
   const headers = request instanceof HonoRequest ? request.raw.headers : request.headers
-  const contentType = headers.get('Content-Type')
+  const contentType = headers.get('Content-Type')!
 
-  if (isFormDataContent(contentType)) {
+  if (
+    (contentType !== null && contentType.startsWith('multipart/form-data')) ||
```

Comment:
Is the below better to avoid using `!`?

```ts
  const contentType = headers.get('Content-Type')

  if (
    (contentType !== null && contentType.startsWith('multipart/form-data')) ||
    (contentType !== null && contentType.startsWith('application/x-www-form-urlencoded'))
  ) {
    return parseFormData<T>(request, options)
  }
```

Reply:
Yes. I will change it